### PR TITLE
Remove references to TTAs providing statement of comparability

### PR DESCRIPTION
**Pull request by MylesJarvis**
We say that TTAs can do statement of comparability. This is not true. Amending the copy to reflect this based on suggestions from colleagues in Candidate Experience.

### CODE REVIEW

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 1]
```diff
@@ -78,7 +80,11 @@ You usually need the equivalent of a GCSE (grade 4) in English and maths.
 
 To teach children aged 3 to 11, you also need the equivalent of a GCSE (grade 4) in a science subject.
 
-[Contact UK ENIC to find out if your qualifications meet the standard](https://www.enic.org.uk/).
+#### To make sure your qualifications are equivalent
```
Comment (peteryates) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:83]:
```suggestion
#### Making sure your qualifications are equivalent
```

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 2]
```diff
@@ -78,7 +80,11 @@ You usually need the equivalent of a GCSE (grade 4) in English and maths.
...
+
+If all or part of your qualifications are from outside the UK, you should contact the schools or universities you would like to train with, as they make the final decision if your qualifications meet the minimum requirements.
+
+For help understanding UK qualifications and their international equivalents, you can call us for free on [0800 389 2500](tel://08003892500). You may be able to get a free statement of comparability, showing how your overseas qualifications compare to UK qualifications. This normally costs £49.50 plus VAT.
```
Comment (peteryates) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:87]: Worth outlining what the criteria for a free one is?

File: app/views/content/train-to-teach-in-england-as-an-international-student.md (hunk 1 above)
Comment (MylesJarvis) [reply app/views/content/train-to-teach-in-england-as-an-international-student.md:83]: This aims to follow on from an earlier header of 'You need:'

**Comment by MylesJarvis** [comment]
@peteryates @Tomcowan1 This is now content proposed by Laura Tennant in International Teacher Recruitment team - reads a lot nicer than mine

**Review by Tomcowan1 — DISMISSED** [review]

**Review by Tomcowan1 — APPROVED** [review]

**Pull request merged**